app/Http/Requests/Album/SetAlbumTagsRequest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Http\Requests\Album;
+
+use App\Http\Requests\BaseApiRequest;
+use App\Http\Requests\Contracts\HasAbstractAlbum;
+use App\Http\Requests\Contracts\HasTagAlbum;
+use App\Http\Requests\Contracts\HasTags;
+use App\Http\Requests\Traits\HasTagAlbumTrait;
+use App\Http\Requests\Traits\HasTagsTrait;
+use App\Models\TagAlbum;
+use App\Rules\RandomIDRule;
+
+class SetAlbumTagsRequest extends BaseApiRequest implements HasTagAlbum, HasTags
+{
+	use HasTagAlbumTrait;
+	use HasTagsTrait;
+
+	/**
+	 * For historic reasons the parameter of the API is called `show_tags`
```

Comment:
```suggestion
	 * For historical reasons the parameter of the API is called `show_tags`
```

But actually, since you are cleaning up the API anyway, wouldn't it make sense to get rid of any such "historical reasons"?